src/commands/zork.js

```diff
@@ -23,9 +25,11 @@ class Zork extends BaseCommand {
       this.logger.warn('Standby, loading the dungeon...');
 
       try {
-        await this.exec('npm install --no-save zorkjs');
+        this.plugins = this.plugins || new Plugins(this.config);
+        await this.plugins.install('zorkjs', { tag: 'latest', force: false });
```

Comment:
Got this the first time:
```
➜  rm -rf ~/.twilio/node_modules/zorkjs 
➜  bin/run zork -l debug               
[DEBUG] Config File: /Users/sharrison/.twilio/config.json
 » Standby, loading the dungeon...
[DEBUG] {}
 » I don't know the word "zork".
```

Works the second time, which is why I had to manually delete `zorkjs`.

Reply:
Ahh, same for me. Weird.